### ISSUE TITLE
Automated tests: assert-value fails incorrectly

### ISSUE THREAD

**Issue by simonratner** [NordicSemiconductor/Android-nRF-Connect#34]
I have an `<assert-value value="90330000"/>` check inside an `<assert-notification>`. In the logs, I see the following:
```
Notification received from ..., value: (0x) 90-33-00-00
"(0x) 90-33-00-00" received
Notification received
Assert-Value...
Value of characteristic '90330000' is not equal to ''
```

Any idea why the assertion is failing even though it seems to receive the correct data?

**Comment by simonratner** [comment]
Same seems to be true for `<assert-value>` inside `<read>` and `<read-descriptor>`.

**Comment by simonratner** [comment]
App version 4.11.0.

**Comment by philips77** [comment]
To bad... I'll look into it when get back from vacations.

**Comment by simonratner** [comment]
👍

**Comment by shafiul2s** [comment]
Yeh we need  back to

**Comment by philips77** [comment]
I found the but. Sooo stupid. Will publish an update soon.

**Comment by philips77** [comment]
Fixed in version 4.11.1. Hopefully. Could you verify, please?
